Basic/ClientDriven/Assets/Editor/Tutorials/TableOfContents.asset

```diff
@@ -0,0 +1,73 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 89305aa391d1c5141bbe1628d930a2c5, type: 3}
+  m_Name: TableOfContents
+  m_EditorClassIdentifier: 
+  Modified:
+    m_PersistentCalls:
+      m_Calls: []
+  ParentContainer: {fileID: 0}
+  OrderInView: 0
+  BackgroundImage: {fileID: 0}
+  Title:
+    m_Untranslated: Table of Contents
+  Subtitle:
+    m_Untranslated: Resources to learn more about Netcode for GameObject's APIs
+  Description:
+    m_Untranslated: 
+  ProjectLayout: {fileID: 0}
+  Sections:
+  - OrderInView: 1
+    Heading:
+      m_Untranslated: Dedicated Forum
+    Text:
+      m_Untranslated: Ask questions and get help on Netcode for GameObjects
+    Metadata: 
+    Url: https://forum.unity.com/forums/netcode-for-gameobjects.661/
```

Comment:
This is the link to the old forum, it has a redirect but maybe we should link to discussion directly
https://discussions.unity.com/tags/c/unity-engine/52/netcode-for-gameobjects